src/core/qgsvectorlayer.h

```diff
@@ -803,6 +803,12 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     bool writeStyle( QDomNode& node, QDomDocument& doc, QString& errorMessage ) const override;
 
     bool writeSld( QDomNode& node, QDomDocument& doc, QString& errorMessage ) const;
+
+    /**
+     * @note not available in Python bindings
+     */
+    bool writeSld( QDomNode& node, QDomDocument& doc, QString& errorMessage, QgsStringMap props = QgsStringMap() ) const;
```

Comment:
Sorry, I missed this update. At the beginning I was thinking not to expose it, but after a conversation with @m-kuhn I was suggested to always expose to Python whatever new method I add. So I guess I added it to the python bindings, but then forgot to remove the comment here. I'll fix this shortly.